orga/app/components/team/members-list-item.hbs

```diff
@@ -4,25 +4,28 @@
 
   {{#unless this.isEditionMode}}
     <td>{{this.displayRole}}</td>
-    {{#if (not-eq @membership.user.id this.currentUser.prescriber.id)}}
-      <td class="zone-edit-role hide-on-mobile">
-        <Dropdown::IconTrigger
-          @icon="ellipsis-v"
-          @dropdownButtonClass="zone-edit-role__dropdown-button"
-          @dropdownContentClass="zone-edit-role__dropdown-content"
-          @ariaLabel={{t "pages.team-members.actions.manage"}}
-        >
-          <Dropdown::Item @onClick={{fn this.editRoleOfMember @membership}}>
-            {{t "pages.team-members.actions.edit-organization-membership-role"}}
-          </Dropdown::Item>
-          <Dropdown::Item @onClick={{fn this.displayRemoveMembershipModal @membership}}>
-            {{t "pages.team-members.actions.remove-membership"}}
-          </Dropdown::Item>
-        </Dropdown::IconTrigger>
-      </td>
-    {{else}}
-      <td class="hide-on-mobile"></td>
+    {{#if this.currentUser.isAdminInOrganization}}
+      {{#if (not-eq @membership.user.id this.currentUser.prescriber.id)}}
```

Comment:
**suggestion**
Je sais que c'est hors scope de la PR mais est-ce qu'on pourrait exporter ça dans une variable avec un nom plus parlant ? 
Ex: `{{#if this.isNotCurrentUserMembership)}}` ou encore `{{#unless this.isCurrentUserMembership}}` 
Et dans `members-list-item.js` on aurait :
```
get isCurrentUserMembership() {
   return this.currentUser.prescriber.id === this.args.membership.user.id;
}
```

Reply:
Done